packages/icons/src/__generated__/webstudio-white.tsx

```diff
@@ -0,0 +1,15 @@
+// Generated from icons/webstudio-white.svg
```

Comment:
it should be webstudio-logo-flat from figma https://www.figma.com/file/sfCE7iLS0k25qCxiifQNLE/%F0%9F%93%9A-Webstudio-Library?type=design&node-id=5196-8318&mode=design&t=AinIdzpi2lwzIbSK-4

and color  is either passed directly or using currentColor (color property of a parent)

Reply:
gotta replace the fill on the source with currentColor when adding

Reply:
I think if you delete the fill everywhere except the root svg node, it will work, not sure though, maybe need to set currentColor

Reply:
So I can modify the props in the generated icon.tsx files?

Reply:
I think the solution would be to delete the svg files so it won't generate again and move the generated `.tsx` files to the parent directory and modify the contents.

Reply:
generated folder is readonly, @TrySound  how does one ensure the color prop is replacing all accourances? is using currentColor going to let svgo replace?